docs/pelion_ready/device-management-config.md

```diff
@@ -68,11 +68,13 @@ Edit the `mbed_app.json` file, and create a new entry under `target_overrides` w
             "device-management.sotp-num-sections" : 2
    ```
 
-`*-address` defines the start of the Flash sector, and `*-size` defines the actual sector size. `sotp-num-sections` should always be set to `2`.
+   - `*-address` defines the start of the flash sector.
+   - `*-size` defines the actual sector size. 
+   - `sotp-num-sections` should always be set to `2`.
 
-At this point, we recommend you run the "connect" test suite, which verifies that the device can successfully bootstrap, register and send and receive data from Device Management with the provided configuration.
+At this point, we recommend that you run the "connect" test suite, which verifies that the device can successfully bootstrap, register and send and receive data from Device Management with the provided configuration.
 
-If you already configured your Device Management API key and initialized your credentials as described in the [previous section](#adding-device-management-feature-to-your-application), you can compile the "Connect" tests using:
+If you already configured your Device Management API key and initialized your credentials as described in [Adding Device Management connectivity to your application](../mbed-os-pelion/device-management-for-mbed-os.html#adding-device-management-connectivity-to-your-application), you can compile the "connect" tests as follows:
```

Comment:
If this is in the same page, you don't need the book and page names in the link. Just the #header

Reply:
You know I remove "as follows" wherever I see it, right?

Reply:
You are absolutely right. I already forgot it was on the same page.

Reply:
Then again, I did check that this is not actually on the same page. It is in a different doc, therefore with the relative link.

Reply:
Sorry, my bad

Reply:
- Fair enough. I'll try to remember to stop using "as follows". 
- I remove "please", whenever I see it. We don't need to be that polite in the docs.

Reply:
We're a British company. We say "please" and make our PRs queue properly.

Reply:
We are not a British company. We are Japanese, remember.

Reply:
🤣